cli/login.go

```diff
@@ -274,7 +274,59 @@ func (r *RootCmd) login() *serpent.Command {
 				return nil
 			}
 
+			var userResp codersdk.User
```

Comment:
Nit: This doesn't seem to be used until later, so we could declare it further down.